client/src/components/ProjectCard.js

```diff
@@ -7,6 +7,7 @@ import {
   Typography
 } from '@material-ui/core';
 import PropTypes from 'prop-types';
+const standingProjectPic = 'https://image.freepik.com/free-photo/river-foggy-mountains-landscape_1204-511.jpg'
```

Comment:
We should change project default picture to the one I made specifically for this purpose in the public images folder

Reply:
got it. will do

Reply:
what is the name of the pic? I see ocean, sunset, image not found